pkg/test/framework/components/istio/util.go

```diff
@@ -68,13 +76,86 @@ func waitForValidationWebhook(ctx resource.Context, cluster resource.Cluster, cf
 }
 
 func (i *operatorComponent) RemoteDiscoveryAddressFor(cluster resource.Cluster) (net.TCPAddr, error) {
+	var addr net.TCPAddr
 	cp, err := i.environment.GetControlPlaneCluster(cluster)
```

Comment:
It looks like you only use `cp` in the `else` below. Let's move it there.

Reply:
using 'cp` in both `if` and `else` and the print logic now